setup.sh

```diff
@@ -139,9 +139,11 @@ if [[ -z $QUIET_INTRO ]]; then
         if [[ $OLD_COMMIT != $NEW_COMMIT ]]; then
             # After syncing to remote, delegate to the new setup script
             # ... in case that changed.
-            git --git-dir="$KIT_PATH/.git" --work-tree="$KIT_PATH" checkout --quiet -B adsk-setup && \
+            # [jokram] Why is the checkout of a new branch adsk-setup necessary? The reset --hard will overwrite it, or?
+            # Or is it just to mark that this was the previous version?
+            git --git-dir="$KIT_PATH/.git" --work-tree="$KIT_PATH" checkout --quiet -B $KIT_ID-setup && \
             git --git-dir="$KIT_PATH/.git" --work-tree="$KIT_PATH" reset --quiet --hard origin/$BRANCH && \
```

Comment:
OK. Next try: `git checkout -f -B adsk-setup origin/production`
Documentation for -f says:
> When switching branches, proceed even if the index or the working tree differs from HEAD. This is used to throw away local changes.